WalletWasabi.Tests/UnitTests/Hwi/DefaultResponseTests.cs

```diff
@@ -36,7 +38,7 @@ public void CanCreate(Network network)
 		[Fact]
 		public void ConstructorThrowsArgumentNullException()
 		{
-			Assert.Throws<ArgumentNullException>(() => new HwiClient(null));
+			Assert.Throws<ArgumentNullException>(() => new HwiClient(null!));
```

Comment:
The null forgiving operator is an antipattern.

Reply:
If you want to test, how `HwiClient` behaves when you pass `null`, you need to pass `null`. 

Personally, I would delete that test because one is not supposed to pass `null` in that class at all.

Reply:
We can remove the test (and the Guard) if we make sure `HwiClient` doesn't get null anywhere, so we can count on that nobody creates new warnings.

Otherwise keeping the warning is preferable than hacking it around.